tests/contest/contest/src/tests/cgroups/cpu/v1.rs

```diff
@@ -219,6 +223,81 @@ fn test_cpu_cgroups() -> TestResult {
     TestResult::Passed
 }
 
+fn check_cgroup_subsystem(
+    cgroup_name: &str,
+    subsystem: &str,
+    filename: &str,
+    expected: &dyn ToString,
+) -> Result<()> {
+    let cgroup_path = Path::new(CGROUP_ROOT)
+        .join(subsystem)
+        .join("runtime-test")
+        .join(cgroup_name)
+        .join(filename);
+
+    debug!("reading value from {:?}", cgroup_path);
+    let content = fs::read_to_string(&cgroup_path)
+        .with_context(|| format!("failed to read {cgroup_path:?}"))?;
+    let trimmed = content.trim();
+    assert_eq!(trimmed, expected.to_string());
+    Ok(())
+}
+
+fn test_relative_cpus() -> TestResult {
+    let case = test_result!(create_cpu_spec(
+        1024,
+        100000,
+        50000,
+        None,
+        "0-1",
+        "0",
+        get_realtime_period(),
+        get_realtime_runtime(),
+    ));
+    let spec = test_result!(create_spec("test_relative_cpus", case.clone()));
+
+    test_outside_container(spec, &|data| {
+        test_result!(check_container_created(&data));
+        test_result!(check_cgroup_subsystem(
+            "test_relative_cpus",
+            "cpu,cpuacct",
+            "cpu.shares",
+            &test_result!(case.shares().context("no shares value in cpu spec")),
+        ));
+        test_result!(check_cgroup_subsystem(
+            "test_relative_cpus",
+            "cpu,cpuacct",
+            "cpu.cfs_period_us",
+            &test_result!(case.period().context("no period value in cpu spec")),
+        ));
+        test_result!(check_cgroup_subsystem(
+            "test_relative_cpus",
+            "cpu,cpuacct",
+            "cpu.cfs_quota_us",
+            &test_result!(case.quota().context("no period value in cpu spec")),
+        ));
+        test_result!(check_cgroup_subsystem(
+            "test_relative_cpus",
+            "cpuset",
+            "cpuset.cpus",
+            &test_result!(case
+                .cpus()
+                .to_owned()
+                .context("no period value in cpu spec"))
+        ));
+        test_result!(check_cgroup_subsystem(
+            "test_relative_cpus",
+            "cpuset",
+            "cpuset.mems",
+            &test_result!(case
+                .mems()
+                .to_owned()
+                .context("no period value in cpu spec")),
```

Comment:
We should be able to unwrap here right? We are making the spec ourselves, so we know these values will always exist?

Reply:
I've fixed it. Btw, just for curiosity. Is `unwrap` considered a bad practice due to panic? And we are okay to put here because of following reasons.

1. We control the output value
2. The test is okay to panic

Reply:
In this particular case it is primarily because we are controlling the output value. As we are building the `case` , we know that those fields will always be present, so we can be sure that this wont panic. While generally it is ok for a test to panic, which indicates a test failure anyways, it is preferable to avoid panicing in tests if possible.